packages/cache-handler/src/handlers/redis-stack.ts

```diff
@@ -146,7 +146,7 @@ export default async function createHandler({
             const keysToDelete: string[] = [];
 
             while (true) {
-                const { documents } = await client.ft.search(
+                const documentIds: Array<string> = await client.ft.searchNoContent(
```

Comment:
`searchNoContent` actually returns an object with this shape:

```ts
interface SearchNoContentReply {
    total: number;
    documents: Array<string>;
};
```

Consider using destructuring assignment:

```suggestion
                const { documents } = await client.ft.searchNoContent(
```